views/otherProfile.hbs

```diff
@@ -0,0 +1,57 @@
+<script src="/js/gallery.js"></script>
+
+{{> nav}}
+<!--Profile picture & biography-->
+<div id = "profileHead" class = "container-fluid padding">
+	<div id = "profileHeader" class = "row welcome text-center fixed-wrap">
+        <div class = "col-12">
+            <button id = "follow" class = "btn btn-outline-dark fluid" style = "position: absolute; 
+                top: 25px;
+                right: 25px;
+                ">Follow</button>
+            <img src = {{user.profilePic}} class = "img-fluid rounded-circle profilePic" alt = "Can't find profile pic">
+         
+        </div>
+		<div class = "col-12">
+            <h1 style = "color:black;" class = 'display-4'>{{user.firstName}} {{user.lastName}}</h1>
+            <h4 class = "secondary" style = "color: #9F9D9D;">@{{user.username}}</h4>
+            <p class = "lead" style = "color:black;">{{user.bio}}</p>
+        </div>        
+    </div>
+    <hr class = "light">
+</div>
+  <!--Contact info-->  
+  <div class = "text-center">
+    <button class = "btn btn-dark" data-toggle = "collapse" data-target= "#info" style = "width: 100%; margin: 0;"><i style="color: white;">Contact Info</i></button>
+    </div>
+
+<div id = "info" class = "collapse">
+  <div class = "container-fluid padding">
+    <div class = "row text-center">
+		<div class = "col-6">
+         <p class="text-info">Email: {{user.email}}</p>
+            <p class="text-info">Mobile Number: {{user.mobile}}</p>
+        </div>        
+        <div class = "col-6">
+            <p class="text-info">Telephone Number: {{user.telephone}}</p>
+            <p class="text-info">Address: {{user.address}}</p>
+        </div>   
+    </div>
+  </div>
+</div>
+
+<hr class = "light" style = "margin: 0;">
+
+<!--following/followers-->
+<div class = "container-fluid padding">
+    <div class = "row">
+		<div class = "col-6 text-right">
+            <p class = "secondary" style = "color: #9F9D9D;">{{#count user.followers}}{{/count}} followers</p>
+        </div>        
+        <div class = "col-6 text-left">
+            <p class = "secondary" style = "color: #9F9D9D;">{{#count user.followings}}{{/count}} following</p>
+        </div>   
+    </div>
+</div>
+
+<div class = "gallery"></div>
```

Comment:
This is a duplicate of "profile.hbs"

Reply:
This is done to differentiate the logged in user profile and the other users (views). This would be removed only if the logged in user is already implemented as the follow button or editprofile button can be filtered using if else with the req.session.user